src/main/java/uk/nhs/digital/nhsconnect/lab/results/model/edifact/DiagnosticReportDateIssued.java

```diff
@@ -0,0 +1,64 @@
+package uk.nhs.digital.nhsconnect.lab.results.model.edifact;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NonNull;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.message.EdifactValidationException;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.message.Split;
+import uk.nhs.digital.nhsconnect.lab.results.utils.TimestampService;
+
+/**
+ * Example DTM+ISR:202001280957:203'
+ */
+@EqualsAndHashCode(callSuper = false)
+@Getter
+@Builder
+public class DiagnosticReportDateIssued extends Segment {
+
+    private static final String KEY = "DTM";
+    private static final String QUALIFIER = "ISR";
+    private static final String KEY_QUALIFIER = KEY + PLUS_SEPARATOR + QUALIFIER;
+    private static final String DATE_FORMAT = "203";
+
+    private final @NonNull LocalDateTime dateIssued;
```

Comment:
In my experience, annotations like this are preferred on the field rather than the type, but I can't find anything explicitly advising this. My best argument then is consistency within our library. where fields, methods and classes are typically annotated one-per-line above the declaration:
```suggestion
    @NonNull
    private final LocalDateTime dateIssued;
```

Reply:
Yeah I agree with you on this. 

I was copying the style used for some segments in NHAIS, but using annotations above fields would be consistent in our project.